src/Test/Utilities/DiagnosticDescription.cs

```diff
@@ -192,6 +192,8 @@ public DiagnosticDescription WhereSyntax(Func<SyntaxNode, bool> syntaxPredicate)
             return new DiagnosticDescription(_code, _isWarningAsError, _squiggledText, _arguments, _startPosition, syntaxPredicate, _argumentOrderDoesNotMatter, _errorCodeType);
         }
 
+        public object Code => _code;
```

Comment:
This looks like a public API change. Did it go through the proper approval process? If it was approved, please exclude it from this pull request and submit as a separate change.